readme-vars.yml

```diff
@@ -71,6 +71,7 @@ app_setup_block: |
 
 # changelog
 changelogs:
+  - { date: "13.01.21:", desc: "Update to Ubuntu Focal base"}
```

Comment:
Link to the FAQ of what this means for ARMHF users, as seen add a note in the readme linking to the FAQ. As seen in https://github.com/linuxserver/docker-domoticz#versions